exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,9 @@
 const name = " Daniel  ";
 
-console.log(message);
+console.log(
+	"My name is " +
```

Comment:
Hi Haroon, this code works correctly, I just want to mention the indentation. I do notice you're using tabs to indent, it's usually better to use spaces, this is because when people use spaces the code will always be indented uniformly across all systems.

When you use tabs, you get to change how it appears, this can cause some minor formatting issues.

I would format this code as such:
```javascript
console.log("My name is " + name.trim() + " and my name is " + name.length + " characters long.");
```

If you do want to split it into multiple lines (usually when the code exceeds the advised character lengths), you could do it like such:
```javascript
console.log(
    "My name is "
    + name.trim()
    + " and my name is "
    + name.length
    + " characters long."
)
```

If you wanted to go the extra mile, you can also use backticks to concatenate, for example:

```javascript
console.log(`My name is ${name.trim()} and my name is ${name.length} characters long.`);
```

Reply:
I will use one space for indenting from now on. thanks. for the formatting, the code is automatically being formatted with Prettier, CYF have told us to use it.